exercises/B-hello-world/exercise.js

```diff
@@ -1 +1,7 @@
-console.log("Hello world");
+console.log("Hello world, I am frustrated!!!");
+console.log("This is my second string")
+let number
+console.log('number')
+console.log(3434)
+console.log("The third line will be an error")
```

Comment:
Just a minor correction: The third line would be undefined but not an error. It is a valid JavaScript to declare a variable and later assign a value to it like:
```
let number;
number = 33;
```
Also if you were trying to console log the third line in line number 4, you don't need to wrap it in a quotation as it is a variable. You can simply do console.log(number) which will print undefined in the console but console.log('number') gives you number as a string value.